benchmark_runner/common/clouds/BareMetal/bare_metal_operations.py

```diff
@@ -92,6 +93,24 @@ def _wait_for_install_complete(self, sleep_time: int = SHORT_TIMEOUT):
             current_wait_time += sleep_time
         raise OCPInstallationFailed(install_log)
 
+    def _wait_for_upgrade_complete(self, oc: OC, sleep_time: int = SHORT_TIMEOUT):
```

Comment:
How long is the short timeout?

Reply:
[600 sec](https://github.com/redhat-performance/benchmark-runner/blob/a1fef4f7b6779e033c464f36933389cd5ef84c20/benchmark_runner/common/clouds/BareMetal/bare_metal_operations.py#L23)

Reply:
Think that will be long enough?

Reply:
Yes, its just the sleep wait in the loop.
The total timeout is [10800](https://github.com/redhat-performance/benchmark-runner/blob/4d50568db561651aa9836b6f3816e81b5da826d6/benchmark_runner/common/clouds/BareMetal/bare_metal_operations.py#L84)